data-prepper-plugins/aws-lambda/src/main/java/org/opensearch/dataprepper/plugins/lambda/processor/LambdaProcessorConfig.java

```diff
@@ -37,6 +40,9 @@ public class LambdaProcessorConfig {
     @JsonProperty("payload_model")
     private String payloadModel = BATCH_EVENT;
 
+    @JsonProperty("sdk_timeout")
```

Comment:
You should add the `@JsonPropertyDescription` annotation to all of these config parameters.

Reply:
Why do you call this `sdk_timeout`? This seems more related to the implementation than what is actually timing out. We should probably look at using `socket_timeout` or `connect_timeout`.